Source/WebCore/dom/EventLoop.h

```diff
@@ -119,16 +121,19 @@ class EventLoop : public RefCounted<EventLoop>, public CanMakeWeakPtr<EventLoop>
     void addAssociatedContext(ScriptExecutionContext&);
     void removeAssociatedContext(ScriptExecutionContext&);
 
+    void invalidateNextTimerFiringTimeCache() { m_nextTimerFiringTimeCache = std::nullopt; }
```

Comment:
"firing time" -> "fire time" for these two names.

Reply:
Fixed.